assets/src/components/progress-bar/style.scss

```diff
@@ -19,6 +19,7 @@
 	overflow: hidden;
 	position: relative;
 	width: calc(100% - 8px);
+	transform: scale(1); // This prevents overflow bug in Safari.
```

Comment:
Note: this it not directly related to this issue. I've fixed is as I noticed some issues on Safari when testing. See: [d325684](https://github.com/ampproject/amp-wp/pull/6835/commits/d32568494e48b27c55ad8fd22a58ca4cc14cc4dc).